packages/snapshots/differ.js

```diff
@@ -0,0 +1,20 @@
+const PNG = require("pngjs").PNG;
+const pixelmatch = require("pixelmatch");
+
+module.exports = (img1, img2) => {
+  const diff = new PNG({ width: img1.width, height: img1.height });
+  const diffCount = pixelmatch(
+    img1.data,
+    img2.data,
+    diff.data,
+    img1.width,
+    img1.height,
+    {
+      threshold: 0.1
```

Comment:
Be cool to have this set from config, maybe for a v2

Reply:
good shout for a v2, I'll log an issue